packages/trader/src/AppV2/Components/PayoutInfo/payout-info.tsx

```diff
@@ -0,0 +1,17 @@
+import { Localize } from '@deriv/translations';
+import { TContractInfo } from '@deriv/shared';
+import { Text } from '@deriv-com/quill-ui';
+import CardWrapper from 'AppV2/Components/CardWrapper';
+import React from 'react';
+
+interface ContractInfoProps {
+    contract_info: TContractInfo;
+}
+
+const PayoutInfo = ({ contract_info }: ContractInfoProps) => (
+    <CardWrapper title={<Localize i18n_default_text='How do I earn a payout?' />}>
+        <Text size='sm'>{contract_info.longcode}</Text>
```

Comment:
![image](https://github.com/binary-com/deriv-app/assets/56330681/f102766a-2937-4ff5-a7f6-8eecf50b4c64)
the content for multipliers seems to be different in the design

Reply:
This comes from API, it needs to be updated by BE team I guess